src/test/java/com/kryptnostic/datastore/edm/DatastoreTests.java

```diff
@@ -265,7 +276,7 @@ public void testAddExistingPropertyTypeToEntityType() {
         dms.addPropertyTypesToEntityType(ENTITY_TYPE.getNamespace(), ENTITY_TYPE.getName(), properties);
     }
     
-    @Test(expected=BadRequestException.class)
+    @Test(expected=IllegalArgumentException.class)
```

Comment:
You shouldn't have the expected method if you are catching and handling this exception below.